demos/demo-minimal-js/index.html

```diff
@@ -90,6 +90,7 @@
         <input type="button" value="log in" id="login" disabled />
         <input type="button" value="log out" id="logout" disabled />
         <div class="step">3. Start a call</div>
+        <input type="button" value="incoming call started" id="incomingcall" disabled />
         <input type="button" value="outgoing call started" id="startcall" disabled />
```

Comment:
For consistency, should we rename this id to `outgoingcall` and startCall() => outgoingCall()?